libfreerdp/gdi/gdi.c

```diff
@@ -562,17 +564,20 @@ static void gdi_set_bounds(rdpContext* context, rdpBounds* bounds)
 	{
 		gdi_SetNullClipRgn(gdi->drawing->hdc);
 	}
+	return TRUE;
 }
 
-static void gdi_dstblt(rdpContext* context, DSTBLT_ORDER* dstblt)
+static BOOL gdi_dstblt(rdpContext* context, DSTBLT_ORDER* dstblt)
 {
 	rdpGdi* gdi = context->gdi;
 
-	gdi_BitBlt(gdi->drawing->hdc, dstblt->nLeftRect, dstblt->nTopRect,
-			dstblt->nWidth, dstblt->nHeight, NULL, 0, 0, gdi_rop3_code(dstblt->bRop));
+	if (gdi_BitBlt(gdi->drawing->hdc, dstblt->nLeftRect, dstblt->nTopRect,
+			dstblt->nWidth, dstblt->nHeight, NULL, 0, 0, gdi_rop3_code(dstblt->bRop)) != 1)
```

Comment:
afaik return value 0 means failure and anything else is success. if that's true then I'd suggest to change this to == 0